### ISSUE TITLE
Simple GP Regression example fails

### ISSUE THREAD

**Issue by NMUrban** [cornellius-gp/gpytorch#525]
The tutorial notebook for Simple GP Regression fails here:
```
print('Iter %d/%d - Loss: %.3f   log_lengthscale: %.3f   log_noise: %.3f' % (
    i + 1, training_iter, loss.item(),
    model.covar_module.base_kernel.log_lengthscale.item(),
    model.likelihood.log_noise.item()
))
```
Specifically, `model.covar_module.base_kernel.log_lengthscale.item()` gives:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/anaconda3/lib/python3.6/site-packages/gpytorch/module.py in __getattr__(self, name)
    224             try:
--> 225                 return super().__getattribute__(name)
    226             except AttributeError:

AttributeError: 'RBFKernel' object has no attribute 'log_lengthscale'
```

and `model.likelihood.log_noise.item()` gives:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/anaconda3/lib/python3.6/site-packages/gpytorch/module.py in __getattr__(self, name)
    224             try:
--> 225                 return super().__getattribute__(name)
    226             except AttributeError:

AttributeError: 'GaussianLikelihood' object has no attribute 'log_noise'
```

**Comment by gpleiss** [comment]
This is just a small issue with the print statements. I'll push a fix.

**Comment by gpleiss** [comment]
(In the future, this will be resolved by #358)

**Comment by gpleiss** [comment]
Should be working now.
